### PR TITLE
HMA-4013

### PR DESCRIPTION
**Pull request by chrisob55**
https://jira.tools.tax.service.gov.uk/browse/HMA-4013

* Updating kotlin multiplatform plugin
* Updating `klock` version
* Adding Swift PM support
* Adding `fastlane` tag release process

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Change Log
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/)
+and this project adheres to [Semantic Versioning](http://semver.org/).
+
+## [Unreleased]
```
Comment (georgeherby) [inline CHANGELOG.md:7]: Does this need the newest version in it?

Comment (chrisob55) [reply CHANGELOG.md:7]: I'll add "Added support for Swift Package Manager" when I tag 0.5.0

File: fastlane/Fastfile [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+#
+
+# Uncomment the line if you want fastlane to automatically update itself
+# update_fastlane
+
+fastlane_require 'semantic'
+
+default_platform(:android)
+
+platform :android do
```
Comment (lojbrooks) [inline fastlane/Fastfile:20]: Not sure what difference it makes, but is `android` the correct platform to use here?

Comment (chrisob55) [reply fastlane/Fastfile:20]: Pretty sure we need to nominate one platform, but in this case it doesn't matter as it's just for the tagging

**Review by lojbrooks — APPROVED** [review]

**Pull request merged**